### ISSUE TITLE
Getting Logits from Cellpose Prediction

### ISSUE THREAD

**Issue by AkshitSharma1** [MouseLand/cellpose#715]
I am using Cellpose to make a prediction. Is there any way to get the logits for the cellpose mask predicted?

**Comment by mrariden** [comment]
Yes - you can do this using the regular API method [with an example here](https://cellpose.readthedocs.io/en/latest/notebook.html). The API for the `models.Cellpose.eval()` call [is documented here](https://cellpose.readthedocs.io/en/latest/api.html#cellpose.models.Cellpose.eval). The returned `flows` contains the information you're looking for.
